addons/mail/static/src/messaging/entity/activity/activity.js

```diff
@@ -225,9 +250,24 @@ function ActivityFactory({ Entity }) {
         }),
         icon: attr(),
         id: attr(),
+        isCurrentPartnerAssignee: attr({
+            compute: '_computeIsCurrentPartnerAssignee',
+            default: false,
+            dependencies: [
+                'assignee',
```

Comment:
This dependency is implicit with assigneePartner, I don't think it needs to be here.

Reply:
I copied what is done in `thread.js` as adviced by AKU, we should change both then ?

```
isCurrentPartnerFollowing: attr({
    compute: '_computeIsCurrentPartnerFollowing',
    default: false,
    dependencies: [
        'followers',
        'followersPartner',
        'messagingCurrentPartner',
    ],
}),
```

Reply:
You can try to remove the other too and see if everything is green.

Reply:
Seems to be ok without them